pkg/ipfailover/keepalived/generator.go

```diff
@@ -144,9 +155,10 @@ func GenerateDeploymentConfig(name string, options *ipfailover.IPFailoverConfigC
 	podTemplate := &kapi.PodTemplateSpec{
 		ObjectMeta: kapi.ObjectMeta{Labels: selector},
 		Spec: kapi.PodSpec{
-			HostNetwork: true,
-			Containers:  containers,
-			Volumes:     generateVolumeConfig(),
+			HostNetwork:  true,
+			NodeSelector: generateNodeSelector(name, selector),
```

Comment:
The option/flag is called ```--selector``` on both the router and ipfailover commands.
The last checkin added the selector flag to the router command.  So both those two are consistent. 

The router additionally has a ```--labels``` - whose usage is primarily to uniquely identify a router and its components.